js/packages/web/src/components/ArtContent/index.tsx

```diff
@@ -49,16 +51,22 @@ const CachedImageContent = ({
   preview?: boolean;
   backdrop: string;
 }) => {
-  const { cachedBlob } = useCachedImage(uri || '');
-  const [loaded, setLoaded] = useState(false);
+  const { cachedBlob, isLoading } = useCachedImage(uri || '');
```

Comment:
This uses the loading from the cache image hook instead of Image component from ant.